Pikipick/Pikipick/Extensions/UIScreen+.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  UIScreen.swift
+//  Pikipick
+//
+//  Created by Jun.Mac on 2022/07/28.
+//
+
+import SwiftUI
+
+extension UIScreen{
```

Comment:
```suggestion
extension UIScreen {
```

so,, 사소,,,하지만 스페이스는 소중해요 ..!
이런것 하나로 코드의 완성도가 크게 차이날 수 있답니다 🙏  
로운이 열심히 쓴 코드가 이왕이면 높은 퀄리티로 모두가 읽어줬으면 좋겠어요 !! 😎

Reply:
사so한 완성도까지 신경쓰는 꼼꼼한 로운씨 되겠습니다.👍 감사합니다.